.github/workflows/tox.yml

```diff
@@ -12,8 +12,17 @@ jobs:
               python-version: 3.6
             - environment: py38
               python-version: 3.8
+            - environment: py310
+              python-version: 3.10
             - environment: pep8
               python-version: 3.8
+            - environment: pep8
+              python-version: 3.10
+          exclude:
+            - python-version: 3.10
+              if: ${{ github.ref || github.base_ref != 'refs/heads/stackhpc/zed' }}
```

Comment:
I wonder what the precedence is here

Reply:
@jackhodgkiss may be able to provide some pointers on how to test changes to these workflows. Maybe that should be written down somewhere (README?)

Reply:
These workflows could be tested with https://github.com/nektos/act

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#operators It may be random but the list of the logical operators in the docs starts with brackets and ends with `||` which might imply the order. I'll try out that workflow tester to see how it works though. Might be useful in the future